tests/run_tests.sh

```diff
@@ -5,14 +5,15 @@ VENV_DIR=$BUILD_DIR/test_venv
 INVENTORY=$BUILD_DIR/inventory
 
 function before_run() {
+   if [ "$VIRTUAL_ENV" != "" ] ; then
+      echo "You currently have a virtual env active. Please deactivate it before proceeding."
```

Comment:
What are you protecting against by issuing a message instead of running `deactivate` for the caller?